### DISCUSSION TITLE
Starlette example does not work

### DISCUSSION THREAD

**Discussion by kumarasinghe** [idom-team/idom#1185] [Problem]
### The example [here](https://reactpy.dev/docs/guides/getting-started/running-reactpy.html#running-with-starlette) does not work
<img width="646" alt="Screenshot 2024-01-10 at 7 43 48 PM" src="https://github.com/reactive-python/reactpy/assets/6898293/a8259c92-3c7c-4bd1-805a-c6124b4fb37a">

I believe `from starlette.applications import Starlette` is missing.

**Comment by Archmonger** [comment]
Are you interested in doing a quick PR to fix this?
